extensions/data-workspace/src/services/workspaceService.ts

```diff
@@ -23,7 +23,7 @@ export class WorkspaceService implements IWorkspaceService {
 	private openedProjects: vscode.Uri[] | undefined = undefined;
 
 	constructor() {
-		this.getProjectsInWorkspace(undefined, true);
+		this.getProjectsInWorkspace(undefined, true).catch(err => console.error('Error initializing projects in workspace ', err));
```

Comment:
not surfacing the error? loc?

Reply:
I'm generally not looking to change behavior here, just adding console logging if I thought that an error occurring would be something we'd be interested in seeing in the logs. (I don't expect this case to actually happen, and if it did happen there isn't really anything a user could do about it anyways)